targets/target-mbed/Makefile.mbed

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# use TAB-8
+
+CURDIR          = `pwd`
+TYPE            = release
+TARGET_LIST     = k64f stm32f4 stm32f429i nucleo
+JERRYHEAP      ?= 16
+
+ifneq ($(filter $(board), $(TARGET_LIST)),)
+	TARGET        = $(board)
+ifeq ($(TARGET), k64f)
+	YOTTA_TARGET  = frdm-k64f-gcc
+	TARGET_DIR   ?= /media/$(USER)/MBED
+else ifeq ($(TARGET), stm32f4)
+	YOTTA_TARGET  = stm32f4-disco-gcc
+else ifeq ($(TARGET), stm32f429i)
+	YOTTA_TARGET  = stm32f429i-disco-gcc
+else ifeq ($(TARGET), nucleo)
+	YOTTA_TARGET  = st-nucleo-f401re-gcc
+	TARGET_DIR   ?= /media/$(USER)/NODE_F401RE
+endif
+
+UPPERC_TARGET = $(shell echo $(TARGET) | tr a-z A-Z) 
+INTERM        = build/obj-mbed-$(TARGET)
+OUTPUT        = build/bin/$(TYPE).mbed$(TARGET)
+COPYTARGET    = targets/target-mbed/libjerry
+
+else
+  $(error This board ($(board)) is not supported!)
+endif
+
+EXT_CFLAGS := -D__TARGET_MBED_$(UPPERC_TARGET)
+EXT_CFLAGS += -mlittle-endian -mthumb -mcpu=cortex-m4
+EXT_CFLAGS += -Wno-error=format=
+
+EXT_PORT_DIR := source/
```

Comment:
The "js/" directory does not couse problem. The js2c python script int the "../tools/" directory creates a C file from the "js/" directory content. We do not use this dirctory in the jerry or yotta building process.

Yes, I get some undefined reference errors if  the "port/" is not under the "source/" directory.
I get this: https://gist.github.com/knightburton/9237699e7a4f427acc94200b171d117a
but if the "port/" is under the "source/" it is works fine.